### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by ElianG2007** [farag2/Sophia-Script-for-Windows#420] [Q&A] [answered]
how can i find the file to download windows 10 22h2?

**Comment by farag2** [comment, marked as answer]
![изображение](https://user-images.githubusercontent.com/10544660/206870460-04ca9280-60ce-4f4d-945e-1037bf4051b4.png)
![изображение](https://user-images.githubusercontent.com/10544660/206870475-a5f7e2f0-a2bf-4196-9692-9f857e41d804.png)

What's the problem to download `Sophia.Script.for.Windows.10.v5.14.4.zip`?

**Comment by ElianG2007** [comment]
I can't find the file, is it possible that you can send it to me?

**Comment by farag2** [comment]
https://github.com/farag2/Sophia-Script-for-Windows/releases/latest

and here?

**Reply by ElianG2007** [reply]
yes thank you very much

**Reply by farag2** [reply]
All you needed is to read carefully the main page. There is a HUGE button "DOWNLOAD"!
